### PR TITLE
Update protal docs

### PR DESCRIPTION
**Pull request by AlexZhenWang**

### CODE REVIEW

File: portal/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Contributing to block explorer web portal
+
+## Make pull requests
+Before you submitting, please check that you have:
```
[the remark this answers is not in the document]
Comment (AlexZhenWang) [reply portal/CONTRIBUTING.md]: Thank you, Bart. will change it now

File: portal/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- [ ] Linked relative Github issue and bounty program link in PR description;
+- [ ] Squashed commits into 1 commit before making a PR to keep clean git history;
```
Comment (nickjohngray) [inline portal/CONTRIBUTING.md]: use the word one not 1

Comment (AlexZhenWang) [reply portal/CONTRIBUTING.md]: Okay. Thanks Nick

File: portal/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- [ ] Labelled `PR is ready to review` when all the above items have all been ticked;
+- [ ] Make sure your commit works on Safari, Chrome and Firefox
+- [ ] Make sure your commit works on mobile(ios and android) and desktop
+
+Thank you!
+
+# Set up development environment
+
+1. Clone the repo locally, via `git clone git@github.com:cennznet/explorer.git <optional local path>`
+2. Change directory to the portal project, via `cd portal`
```
Comment (nickjohngray) [inline portal/CONTRIBUTING.md]: is this needed?

Comment (AlexZhenWang) [reply portal/CONTRIBUTING.md]: yeah

File: portal/README.md [hunk 4]
```diff
@@ -1,3 +1,15 @@
...
+
+Contributions are welcome!
+
+To get started -
+
+1. Clone the repo locally, via `git clone git@github.com:cennznet/explorer.git <optional local path>`
+2. Change directory to the portal project, via `cd portal`
+3. Install the dependencies by running `yarn`
+4. Run in the dev mode, via `yarn run dev`
+5. Access the app via [http://localhost:3000](http://localhost:3000)
```
Comment (nickjohngray) [inline portal/README.md]: 1.  git clone git@github.com:cennznet/explorer.git <optional local path> : clone the repo locally
2.  cd portal : change to the portal project directory
3.  yarn : Install the dependencies
4.  yarn run dev: run in dev mode
5.  [http://localhost:3000](http://localhost:3000) : open the app

**Pull request merged**